mobileClient/components/WalletsList.js

```diff
@@ -192,80 +183,79 @@ export default class WalletsList extends React.Component {
   };
 
   render() {
+    const coinAvatarUrls = {
```

Comment:
could we pull the coinAvatars into the assests directory and import `coinAvatarUrls` in?  Or does it need to be inlined or hard coded in?  I think I am using the term hard coded correctly.

Reply:
That should work.

Reply:
I would think static icons are all in an assets folder.  Only user avatars would be stored from a cdn.  But ya go head and merge it now if ya want with or without the `coinAvatarUrls` inlined.

Reply:
I tried and couldn't get it to work.

When importing a local static image, it needs to be required like this
`avatar: {{ uri: require('/path/to/image.png') }}`

For some reason require keeps returning a number instead of the image.